app/forms/PensionDetails_no_longer_used_Form.scala

```diff
@@ -18,15 +18,15 @@ package forms
 
 import com.google.inject.Singleton
 import javax.inject.Inject
-import models.PensionDetails
+import models.PensionDetailsScon
 import play.api.data.Form
 import play.api.data.Forms._
 import play.api.i18n.{Messages, MessagesImpl}
 import play.api.mvc.MessagesControllerComponents
 import validation.SconValidate
 
 @Singleton
-class PensionDetailsForm @Inject()(mcc: MessagesControllerComponents) {
+class PensionDetails_no_longer_used_Form @Inject()(mcc: MessagesControllerComponents) {
```

Comment:
same question as further down - can this not be linked to the new stuff?  Will it be changed prior to final merge/release?

Reply:
same answer as above